apps/core/src/edge-login/README.md

```diff
@@ -0,0 +1,5 @@
+# EdgeLogin
+
+`EdgeLoginModule` extends Core with the `/edgeLogin` REST API.
```

Comment:
I think you would want to use `/edge-login` for the endpoint. I believe kebab casing is the typical.

Reply:
Never mind! I looks like it's a typo in the README.

Reply:
Fixed in latest push. I started with camel casing then realized for apis it should be kebab.